lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -5169,9 +5171,16 @@ DifferentiationTask::DifferentiationTask(ADContext &context,
           original, attr, vjpName, AutoDiffAssociatedFunctionKind::VJP);
     attr->setVJPName(vjpName);
   }
+  // If differentiation is triggered by `@differentiable`, we export its symbol
+  // when the original function is public.
+  auto isAssocFnExported =
+      invoker.getKind() ==
+          DifferentiationInvoker::Kind::DifferentiableAttribute ||
```

Comment:
Is it the case that `DifferentiableAttribute` is never used now?

Reply:
Yes, I can send a followup to delete DifferentiableAttribute from everything tomorrow.

Reply:
great